shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs

```diff
@@ -451,17 +456,35 @@ newtype BootstrapAddress crypto = BootstrapAddress
 
 instance NoUnexpectedThunks (BootstrapAddress crypto)
 
+type BootstrapVerKey_ = DSIGN.VerKeyDSIGN Ed25519DSIGN
+
+newtype BootstrapVerKey = BootstrapVerKey {unBootstrapVerKey :: BootstrapVerKey_}
+  deriving (Eq, Generic)
+  deriving newtype (NFData)
+  deriving (Show) via Quiet BootstrapVerKey_
+
+instance NoUnexpectedThunks BootstrapVerKey
+
+instance FromCBOR BootstrapVerKey where
+  fromCBOR = BootstrapVerKey <$> DSIGN.decodeVerKeyDSIGN
+
+instance ToCBOR BootstrapVerKey where
```

Comment:
I copied this logic from the `ToCBOR (VKey ...)` instance. Is there a difference from what newtype deriving would do?

Reply:
I don't believe the underlying type has the instances.